.gitignore

```diff
@@ -5,3 +5,6 @@ npm-debug.log
 .DS_Store
 docs/
 !scripts/docs/
+
+# Ignore Argos CI screenshots.
+functional-tests/screenshots/
```

Comment:
It's here in case someone is running the e2e tests on his development environment. We don't want the screenshots to be commited.